### DISCUSSION TITLE
Ajout d'un message de validation pour la commande drink/glouglou/dr

### DISCUSSION THREAD

**Discussion by Thesmokingllama** [Crownicles/Crownicles#837] [Idées acceptées / déjà implémentées]
Avec la possibilité de stocker plusieurs potions il est devenu plus difficile de savoir où a été placée la potion de son choix, par conséquent, les erreurs sont fréquentes. C'est pourquoi je propose d'ajouter un message rappelant aux joueurs quelle potion ils s'apprêtent à boire et leur demandant confirmation 

![5E28AEAB-E9EB-49DB-B2C5-2464D22E9FC3](https://user-images.githubusercontent.com/80409766/148390840-64a05a00-9d86-4850-9eec-98710666bed7.jpeg)
*voici l'erreur type de la consommation de potion de soin en pensant boire une potion de temps*

**Comment by ghost** [comment]
J'y pensais !
Faire apparaître cette demande de confirmation seulement si deux emplacements de potions ont été déloqués !

**Reply by BastLast** [reply]
en effet

**Comment by BastLast** [comment]
suggestion approuvée
